react.js

```diff
@@ -5,61 +5,84 @@
 
 // 1. Write a React component that prints "I am a component!" Be sure to include all necessary imports, exports, etc.
 
+import React, { Component } from 'react'
+import './App.css';
 
+class Selfaware extends Component{
 
+  render(){
+    return(
+      <div>
+          <p>
+            I am a component!
+          </p>
+      </div>
+    )
+  }
+}
+
+export default Selfaware
 
 
 // 2. Refactor this vanilla javascript loop to a map function. The output should remain the same.
 
 var names = ["Ford Prefect", "Arthur Dent", "Trillian", "Zaphod", "Vogon", "Marvin, the Paranoid Android"]
 
-for(let i=0; i<names.length; i++){
-  console.log(`${names[i]} is ${names[i].length} characters long.`)
-}
+// for(let i=0; i<names.length; i++){
+//   console.log(`${names[i]} is ${names[i].length} characters long.`)
+// }
 
+names.map((v,i) => {
+  console.log(`${names[i]} is ${names[i].length} characters long.`);
+})
 
```

Comment:
${names[i]} can also be written as ${v} - it does the same thing but if you have value you might as well use it.

Reply:
Duh, I'm so silly. I just lazily copied and pasted everything after the for loop bracket. Thank you.